Content.Server/Mind/Filters/TargetObjectiveMindFilter.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Server.Objectives.Components;
+using Content.Shared.Mind;
+using Content.Shared.Mind.Filters;
+using Content.Shared.Whitelist;
+
+namespace Content.Server.Mind.Filters;
+
+/// <summary>
+/// A mind filter that removes minds if you have an objective targeting them matching a whitelist.
+/// </summary>
+/// <remarks>
+/// Used to prevent assigning multiple kill objectives for the same person.
+/// </remarks>
+public sealed partial class TargetObjectiveMindFilter : MindFilter
+{
+    /// <summary>
+    /// A whitelist to check objectives against to remove a mind.
+    /// If null then any objective targeting it will remove minds.
+    /// </summary>
+    [DataField]
+    public EntityWhitelist? Blacklist;
```

Comment:
You use "whitelist" to describe the property above but it's called "blacklist" here, which seems inconsistent.

Reply:
misinput

Reply:
Aight fix it and I'll mergghjghghjn